### ISSUE TITLE
Training Data and Test Data Requirements for Kgpolicy

### ISSUE THREAD

**Issue by Shadz13** [xiangwang1223/kgpolicy#3]
Can you kindly elaborate on the data structure required for the training set and test set using a basic example (i.e. how the data should be setup)?

In addition, kindly cover how the ratings should be set up for implicit feedback in the example?
